README.md

```diff
@@ -1 +1,54 @@
-# mender-demo-artifact
\ No newline at end of file
+Mender Demo Artifact
+=============================================
+
+Mender is an open source over-the-air (OTA) software updater for embedded Linux
+devices. Mender comprises a client running at the embedded device, as well as
+a server that manages deployments across many devices.
+
+This repository contains a demo artifact, which is used during the onboarding process
+and acts as an example of the update capabilities.
+
+![Mender logo](https://mender.io/user/pages/resources/06.digital-assets/mender.io.png)
+
+## Getting started
+
+
+### Onboarding Site
+ 
+This site is served with some device information from the serving device. The device details are populated in the `entrypoint.sh` in the development setup and a similar file should be executed before server start on the updated device.
```

Comment:
The first sentence is a bit weird, maybe something like "This site displays system information from the device itself", or "... from the serving device" (if you really like the serving part). My point is not to use served and device twice each.

Reply:
good point, fixed this and thanks for the review!